satellite/console/service.go

```diff
@@ -187,14 +187,30 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 		return nil, err
 	}
 
+	var customerID []byte
 	userPayments, err := s.store.UserPayments().Get(ctx, authorization.User.ID)
 	if err != nil {
-		return nil, err
+		if err != sql.ErrNoRows {
+			return nil, err
```

Comment:
use user friendly error messages that are declared in const section in service.go